packages/web-frontend/src/reducers.js

```diff
@@ -587,7 +587,7 @@ function measureBar(
 function global(
   state = {
     isSidePanelExpanded: false,
-    overlay: !isMobile() && LANDING,
+    overlay: null,
```

Comment:
We don't always want to show the landing by default in case a user enters a nested url